portfolio/src/main/webapp/style.css

```diff
@@ -8,65 +8,65 @@ html {
 }
 
 .size {
-    width:100%;
-    height: 650px;
+  width: 100%;
+  height: 650px;
 }
 .font {
-    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
 
 .textsize {
-    font-size:50px;
+  font-size: 50px;
 }
 
 .center {
-    text-align: center;
+  text-align: center;
 }
 
 .right {
-    align-content: right;
+  align-content: right;
```

Comment:
Did you mean to specify:  `align-content: end;` ? It looks like `right` isn't a value of `align-content`: https://developer.mozilla.org/en-US/docs/Web/CSS/align-content